unabridged-works-volume-2/original/08-swearing-under-the-gospel.adoc

```diff
@@ -12,7 +12,7 @@ By _Isaac Penington,_ the Younger
 [.section-date]
 1661
 
-WHETHER it be lawful for Christians (who know Christ,
+Whether it be lawful for Christians (who know Christ,
```

Comment:
So this was a weird and somewhat difficult chapter. This line seems like a section summary, but there's an answer immediately following it so I just left it as is.

Reply:
Yeah, agreed.  I'm a bit torn on this one, but I would side with what you did hear, leaving it as is, so let's go with that.